### DISCUSSION TITLE
Welcome to shield-mvvm Discussions!

### DISCUSSION THREAD

**Discussion by MetalKid** [CoreBTS/shield-mvvm#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by peter-custance** [comment]
Hi, I arrived here will looking for a replacement for FreshMvvm in MAUI. I have worked with Xamarin from the day's of Mono and Mono Develop. I have used FreshMvvm in a lot Of Xamarin Apps as I prefer the model to model navigation. When I started looking at MAUI, I found the default workflows clunky and alien to me. I quickly tried the FreshMvvm for MAUI which stopped working after a few versions and by converting it from to a MAUI library it worked again. Being a very busy full time developer in many other types of projects I found I did not have the time to actual maintain this. I started looking for another MAUI Mvvm where I could be a part or contribute which might fit in more with my available time and arrived at shield-mvvm. I can see that this project is offering much of what I loved in FreshMvvm but with the fact that it is active and progressing. I'm developing some new apps in MAUI, both in my own right and also in my day job and look forward to both using and hopefully contributing. I've been a .NET developer since the beginning and started programming in 1993 in VB/C++ here in the UK

**Reply by MetalKid** [reply]
That sounds very interesting! Shield is probably closer to MvvmCross, but only based on a couple method names. I'm still working on getting this released to Nuget as we need to sign the release since it's from my company. I'd love to hear any feedback you have as well. While you could just use this for the navigation part, the most power comes from the code-behind bindings. I'd love to hear your opinion on that side of things, too!

**Comment by peter-custance** [comment]
Yes I use MvvmCross with WPF and noticed the code behind bindings. I tend to create all my content pages in code rather than Xaml. I find this habit hard to break as I can visualise the UX in my mind. It was originally the easiest way in the early days of Xamarin Forms.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by MetalKid** [reply]
@peter-custance FYI: Finally got this released as a nuget package (https://www.nuget.org/packages/CoreBTS.Maui.ShieldMVVM) so feel free to play around with it! :)

**Reply by peter-custance** [reply]
Hi,
Just got back to looking at this as November has been busy with last releases before Xmas. I've used the nu-get package and will start playing with it this week. I'm using Maui on .Net 8 as the previous releases were a little challenging shall we say. I will feedback as time allows. Regards

**Comment by peter-custance** [comment]
Sure Dan, I will take a look as soon as I can.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________
